ecf/scripts/forecast/firewx/jrrfs_firewx_forecast.ecf

```diff
@@ -3,7 +3,7 @@
 #PBS -S /bin/bash
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
-#PBS -l walltime=01:10:00
+#PBS -l walltime=02:00:00
```

Comment:
@lgannoaa Today's 06Z and 12Z forecasts completed in ~01:10:00.  I forget why they were running slowly in our previous testing, but maybe we could change this resource setting to 01:30:00 instead of 02:00:00.  What do you think?

Reply:
Sound good to me.